### DISCUSSION TITLE
found no layout file for "headers" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination

### DISCUSSION THREAD

**Discussion by arungudelli** [thuliteio/doks#903] [Q&A]
I am migrating my personal blog to getDoks theme.. I am able to copy the content.. The site is running successfully.. But getting this warning. 

"found no layout file for "headers" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination"

Is it fine?

**Comment by h-enk** [comment]
Do you have a repo you could share?

**Reply by arungudelli** [reply]
I understood.. I have deleted index.headers file as I am getting CSP errors.. Thanks for responding..
